starport/cmd/module-create.go

```diff
@@ -80,19 +84,24 @@ func createModuleHandler(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	var msg bytes.Buffer
+	fmt.Fprintf(&msg, "\n🎉 Module created %s.\n\n", name)
 	if err := sc.CreateModule(placeholder.New(), name, options...); err != nil {
-
 		// If this is an old scaffolded application that doesn't contain the necessary placeholder
 		// We give instruction to the user to modify the application
 		if err == scaffolder.ErrNoIBCRouterPlaceholder {
 			fmt.Print(ibcRouterPlaceholderInstruction)
 		}
-
-		return err
+		var validationErr validation.Error
+		if errors.As(err, &validationErr) {
+			fmt.Fprintf(&msg, "Can't register module '%s'.\n", name)
```

Comment:
```suggestion
			fmt.Fprintf(&msg, "Can't register module '%s' in app/app.go.\n", name)
```

Reply:
i would rather avoid pasting path manually. in future it will be a part of the validation msg

Reply:
I think it's fine, it's part of this log. Or I would at least precise that we tried to register the module into `app.go` file